pkg/serverless/metrics/metric.go

```diff
@@ -61,8 +62,12 @@ func (c *ServerlessMetricAgent) Start(forwarderTimeout time.Duration, multipleEn
 	// the invocation metric is also generated by Lambda Layers
 	// we want to avoid duplicate metric
 	customerList := config.Datadog.GetStringSlice("statsd_metric_blocklist")
-	config.Datadog.Set("statsd_metric_blocklist", buildMetricBlocklist(customerList))
-
+	// if the proxy is enabled we need to also block the errorMetric
+	if os.Getenv("DD_EXPERIMENTAL_ENABLE_PROXY") == "true" {
```

Comment:
lowercasing the retrieved value could be useful if a customer is setting it to TRUE